index.html

```diff
@@ -3007,11 +3007,13 @@ <h3>Expiration</h3>
       </section>
 
       <section>
-        <h3>Revocation</h3>
+        <h3>Status Revocation</h3>
 
         <p>
-If revocation instructions are present, the <a>credential</a> MUST NOT
-have been revoked.
+If the <code>credentialStatus</code> propery is available the status of the 
```

Comment:
Editorial nits:

* "propery" is misspelled
* Missing comma between "available, the"

Reply:
fixed